packages/template-typescript-minimal/package.json

```diff
@@ -36,7 +36,7 @@
             "node_modules/**/*.*"
         ],
         "ssrShared": [
-            "static/ico/favicon.ico",
+            "static/favicon.ico",
```

Comment:
The path in the file tree for the `favicon.ico` file didn't change. Why are we changing it here?

Reply:
That was unintentional I thought favicon had to be served from the root but that's not the case. Reverting it back to the original path.

Reply:
Moved `favicon .ico` to the root of the `static` folder inside the typescript template since that's the only static asset there. File path has changed inside `template-typescript-minimal`

Reply:
Got it... I thought this was in retail react app.. by bad. Thanks